src/Common/constants.tsx

```diff
@@ -1,6 +1,6 @@
-import keralaLogo from "../../public/images/kerala-logo.png";
+// import keralaLogo from "../../public/images/kerala-logo.png";
```

Comment:
Can you please remove the code in the comments?

Reply:
This code also seems different that what I intended to push, let me double check.

Reply:
Please have a look again @peeyush14goyal.

Reply:
Its resolved ✔